pkg/reconciler/trigger/trigger_test.go

```diff
@@ -449,6 +450,64 @@ func TestReconcile(t *testing.T) {
 						WithTriggerSubscriberResolvedSucceeded(),
 						WithTriggerStatusSubscriberURI(subscriberURI)),
 				}},
+			}, {
+				Name: fmt.Sprintf("%s: Creates everything with ref subscriber different ns", name),
+				Key:  testKey,
+				Objects: []runtime.Object{
+					ReadyBroker(config),
+					makeSubscriberAddressableAsUnstructuredWithNamespace(otherNS),
+					markReady(createQueue(config, false, "")),
+					markReady(createBinding(false, false, "")),
+					NewTrigger(triggerName, testNS, brokerName,
+						WithTriggerUID(triggerUID),
+						WithTriggerSubscriberRef(subscriberGVK, subscriberName, otherNS)),
+					createSecret(rabbitURL),
+					createRabbitMQBrokerConfig(""),
+					createRabbitMQCluster(""),
+				},
+				WantCreates: []runtime.Object{
+					createDispatcherDeployment(false, ""),
+				},
+				WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
+					Object: NewTrigger(triggerName, testNS, brokerName,
+						WithTriggerUID(triggerUID),
+						WithTriggerSubscriberRef(subscriberGVK, subscriberName, otherNS),
+						WithTriggerBrokerReady(),
+						WithTriggerDeadLetterSinkNotConfigured(),
+						WithTriggerDependencyReady(),
+						WithTriggerSubscribed(),
+						WithTriggerSubscriberResolvedSucceeded(),
+						WithTriggerStatusSubscriberURI(subscriberURI)),
+				}},
+			}, {
+				Name: fmt.Sprintf("%s: Creates everything with ref subscriber no ns provided", name),
+				Key:  testKey,
+				Objects: []runtime.Object{
+					ReadyBroker(config),
+					makeSubscriberAddressableAsUnstructured(),
```

Comment:
This function https://github.com/knative-sandbox/eventing-rabbitmq/pull/1147/files#diff-c5ee2c23a8a6dd4c7aa3e6620d57f2bfd00a6b37dd072ade8bfd386dd4952d21L1406 populates a namespace. Therefor the codecoverage is not being covered in the line specified by codecov.

Reply:
the function sets the name space to https://github.com/knative-sandbox/eventing-rabbitmq/pull/1147/files#diff-c5ee2c23a8a6dd4c7aa3e6620d57f2bfd00a6b37dd072ade8bfd386dd4952d21R78